src/openrct2/rct2/RCT2.h

```diff
@@ -593,6 +593,26 @@ union RCT2Sprite
 };
 assert_struct_size(RCT2Sprite, 0x100);
 
+struct RCT2RideRatingCalculationData
+{
+    uint16_t proximity_x;
+    uint16_t proximity_y;
+    uint16_t proximity_z;
+    uint16_t proximity_start_x;
+    uint16_t proximity_start_y;
+    uint16_t proximity_start_z;
+    uint8_t current_ride;
+    uint8_t state;
+    uint8_t proximity_track_type;
+    uint8_t proximity_base_height;
+    uint16_t proximity_total;
+    uint16_t proximity_scores[26];
+    uint16_t num_brakes;
+    uint16_t num_reversers;
+    uint16_t station_flags;
+};
+assert_struct_size(RCT2RideRatingCalculationData, 76);
```

Comment:
Is there a reason that this has to be strict?

Reply:
Yes... this is all used for loading SV6.

Reply:
Yeah nevermind, misread the code slightly. I thought there was another struct specific to s6/s4 so we wouldn't need to worry about packing given we copy field by field.